.travis.yml

```diff
@@ -15,3 +15,4 @@ script:
   - git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
   - make
   - make test
+  - PATH="${PATH}:${PWD}" make -C test
```

Comment:
> ... we need do 'git submodule update --init' first.

Travis [does that for us][1].

[1]: https://travis-ci.org/opencontainers/runtime-tools/jobs/255352371#L224